.chloggen/sqlserver_fix_for_long_running_queries.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. receiver/filelog)
+component: receiver/sqlserver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fixed bug in query metrics collection for long running queries
```

```suggestion
note: Properly collect query metrics for long running queries
```
Nit: I'm trying trying to make it slightly more clear what the bug is, or really, what's changed here. Adding `subtext` could help too if you think it's not clear enough here. We just want to make it clear that query metrics for long running queries were not being collected before, but they are now, right?

---

A workaround existed (configuring a long lookbackTime so it covers the start time of long running queries too) and it could collect most long running queries. just that now we have a fail proof solution that would work in all cases.

But, agree, lets make the note more straightforward.